pydicom/charset.py

```diff
@@ -554,7 +562,7 @@ def _encode_string_parts(value, encodings):
     """
     encoded = bytearray()
     unencoded_part = value
-    best_encoding = None
+    best_encoding: str
```

Comment:
Didn't know this is possible - it's initialized to an empty string, I guess.

Reply:
It declares that the variable exists without giving it a value ([PEP526](https://www.python.org/dev/peps/pep-0526/), [PEP484](https://www.python.org/dev/peps/pep-0484/#type-comments))

Reply:
In hindsight I could've just used `best_encoding: str = None`